qutebrowser/javascript/webelem.js

```diff
@@ -171,14 +203,72 @@ window._qutebrowser.webelem = (function() {
             }
         }
 
+        // Recurse into frames and add them
+        for (let i = 0; i < subelem_frames.length; i++) {
```

Comment:
Bleh, I hoped we could use [for..of](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) here, but nope...